src/components/common/AddNameTranslation.vue

```diff
@@ -9,25 +9,26 @@
         label="Enter Name Translation"
         v-model="nameTranslation"
         :rules="nameTranslationRules"
+        @input="nameTranslation = nameTranslation.toUpperCase()"
       />
 
       <div class="form__btns">
-        <v-btn large color="error" id="btn-remove"
+        <v-btn large color="error" id="name-translation-btn-remove"
```

Comment:
Aligning all of the btn IDs to be the same as the AddNameTranslation.vue in the Edit UI

Reply:
Great, thanks. Did you also verify that the button ids are not used for styling? (ie, search for `#btn-remove`)

Reply:
Yes they were just used for testing, I also doubled checked the UI locally